config/locales/en.yml

```diff
@@ -83,8 +88,12 @@
     privacy_notice: Privacy notice
     school_home: Get devices for your school
     school_can_order_devices: Your school can order devices
-    school_cannot_order_devices: Your school cannot order devices yet
-    school_user_cannot_order_devices: You cannot order devices yet
+    school_cannot_order_devices:
+      title: Your school cannot your full allocation yet
+      can_order_when: 'Your school can order your full allocation when:'
+    school_user_cannot_order_devices:
+      title: You cannot order your full allocation yet
+      can_order_when: 'You can order your full allocation when:'
```

Comment:
Should `can_order_when` be in the `page_titles.` block?

Reply:
I don't think having a `page_titles` block particularly makes sense. We want to put more copy into this file and we should be separating by more logical chunks eg. by page to group things together. Otherwise page titles of the pages will be in this section and all the other content for those pages will be in some other section.

Reply:
This was the way Apply built their titles, and we reused that pattern. I'm not especially attached to it.